### ISSUE TITLE
5.2: docs: col-auto isn't explained by this point

### ISSUE THREAD

**Issue by willfaught** [twbs/bootstrap#36978]
https://github.com/twbs/bootstrap/blob/c3c65911665ab64bdaa15d405db65ee81655dbf3/site/content/docs/5.2/layout/columns.md?plain=1#L281

**Comment by tungphan995** [comment]
Thank you for believing in me.

**Comment by julien-deramond** [comment]
It is explained in the previous [Layout > Grid > Variable width content](https://getbootstrap.com/docs/5.2/layout/grid/#variable-width-content) page. However, it's maybe not so obvious that `.col-auto` is included in "Use `col-{breakpoint}-auto` classes classes to size columns based on the natural width of their content".

**Comment by willfaught** [comment]
Yes, I searched for "col-auto" in that page, and nothing was found, so I assumed it was missing.

**Comment by willfaught** [comment]
In other pages, all classes are explicitly listed, e.g. justify-content-center. Perhaps it would be best to just list all the breakpoint permutations so they get indexed and are searchable with Ctrl/Cmd-F.

**Comment by willfaught** [comment]
Now that I think of it, sometimes I run across a Bootstrap class, and have no idea where to find the doc for it, e.g. py-3. It'd also be nice to have a page in the doc listing every Bootstrap class alphabetically with a link to its page.
